src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import enums.Colors;
+import java.util.Random;
+
 public class ColorSupplier {
+    private int colorIndex;
+
+    public ColorSupplier() {
+        this.colorIndex = new Random().nextInt(Colors.values().length);
+    }
```

Comment:
its not suggested to add colorIndex as class field 
```suggestion
    private final Random random = new Random();
```

Reply:
++